docs/source/reference/index.rst

```diff
@@ -1,147 +1,143 @@
-.. module:: pytorch_pfn_extras
-
 API Reference
 =============
 
 * :ref:`genindex`
 
+Package
+-------
+.. autosummary::
+   :toctree: generated/
+   :recursive:
+   
+   pytorch_pfn_extras
+
+
 Training Loop
 ------------------
 
 Trainer
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. autosummary::
-   :toctree: generated/
 
-   engine.create_trainer
-   engine.create_evaluator
-   handler.BaseLogic
-   handler.Logic
-   handler.BaseHandler
-   handler.Handler
-   runtime.BaseRuntime
-   runtime.PyTorchRuntime
+   pytorch_pfn_extras.engine.create_trainer
```

Comment:
Question: Is it possible to do this without adding `pytorch_pfn_extras.` prefix?
The columns seems too narrow... 😓
https://pytorch-pfn-extras--731.org.readthedocs.build/en/731/reference/index.html#extensions

Reply:
By using currentmodule, it has been identified that it's possible to change the module within a file. Therefore, this will be utilized to remove the prefix.